packages/core/package.json

```diff
@@ -18,6 +18,7 @@
   ],
   "scripts": {
     "start": "yarn build -w",
+    "prebuild": "node -p \"'export const VERSION = \\'' + String(require('./package.json').version) + '\\';'\" > src/version.ts",
```

Comment:
Maybe move this into a separate script file?

Reply:
We can also do something else here: export the version as a constant enum which is actually replaced in the compiled code like this:
```
export const enum VERSION = {
  CURRENT: '0.0.0-dev',
}

console.log(VERSION.CURRENT);
```

which would be compiled to: `console.log('0.0.0-dev');`

We can do it once in the core package, reference the constant enum in the subpackages which would replace everything with static strings.

Reply:
ohh right, let's do it

Reply:
- [ ] create version file in core only